db/migrate/20140716131107_create_category_translation_tables.rb

```diff
@@ -112,16 +112,16 @@ def down
     new_categories = []
     PublicBodyCategory.all.each do |category|
       category.locale = category.translation.locale.to_s
-      FastGettext.default_available_locales.each do |locale|
-        if locale.to_s != category.locale
+      AlaveteliLocalization.available_locales.each do |locale|
+        if locale != category.locale
```

Comment:
But won't it then get [converted to a Symbol](https://github.com/globalize/globalize/blob/v4.0.3/lib/globalize/active_record/translation.rb#L29) on the way back out in the call to `#locale`?